R/qenv-get_warnings.R

```diff
@@ -0,0 +1,41 @@
+#' Get the warnings of qenv object
+#'
+#' @param object (`qenv`)
+#'
+#' @return `character` containing warning information or `NULL` if no warnings
+#' @export
+#'
+#' @examples
+#' data_q <- new_qenv() %>% eval_code("iris_data <- iris")
```

Comment:
Need library(magrittr) for the example and add magrittr to suggests or just do

data_q <- eval_code(new_qenv(), "iris_data <- iris")

Reply:
done.